components/adblock_rust_ffi/Cargo.toml

```diff
@@ -5,7 +5,7 @@ authors = ["Brian R. Bondy <netzen@gmail.com>"]
 edition = "2018"
 
 [dependencies]
-adblock = { version = "0.7.5", default-features = false, features = ["full-regex-handling", "debug-info", "css-validation"] }
+adblock = { version = "0.7.7", default-features = false, features = ["full-regex-handling", "debug-info", "css-validation"] }
```

Comment:
note: this is skipping past 0.7.6, which was released to support https://github.com/brave/brave-core/pull/17368 (still undergoing code review/revisions).

There are no changes in 0.7.6 which require corresponding brave-core changes, so this should be fine to merge ahead of time and the version change can then be omitted from https://github.com/brave/brave-core/pull/17368 can 

cc @DJAndries

Reply:
The relevant change for 0.7.7 is https://github.com/brave/brave-browser/issues/29099